tests/test_errors.py

```diff
@@ -31,3 +36,51 @@ def test_fk_not_exists(self, client, endpoint, kf_id):
         else:
             message = 'Longer than maximum length 11'
             assert message in response['_status']['message']
+
+    def test_uniqueness_constraints(self, client, entities):
```

Comment:
Could you add a test for family relationships? I did a quick test and saw that the table in the error message wasn't quite parsed correctly: 
```
{
  "_status": {
    "code": 400, 
    "message": "could not create .  with (participant1_id, participant2_id, participant1_to_participant2_relation, participant2_to_participant1_relation) = (PT_3X5795SV, PT_0B265MYJ, mother, Child) already exists."
  }
}
``` 
If there was a way to make this a bit more generic, where you don't have to have an explicit test for each desired table, that would be great. Then you could just have this test take in a pytest parameterized list containing tables (sequencing_center, biospecimen_genomic_file, biospecimen_diagnosis, read_group_genomic_file, family_relationship, etc), similar to the other tests we have (see the test above this one - test_fk_not_exists)

Reply:
> If there was a way to make this a bit more generic, where you don't have to have an explicit test for each desired table, that would be great.

I agree. I'll see if I can come up with something that doesn't violate any principles of test validity.